sdk/sdk.go

```diff
@@ -1031,6 +1033,95 @@ func (c *Client) NewManualRateChangeTransaction(
 	return tx, err
 }
 
+func (c *Client) NewAutomaticExecutionsPaymentTransaction(
+	deadline *Deadline,
+	contractKey *PublicAccount,
+	automaticExecutionsNumber uint32,
+	networkType NetworkType,
+) (*AutomaticExecutionsPaymentTransaction, error) {
+	tx, err := NewAutomaticExecutionsPaymentTransaction(
+		deadline,
+		contractKey,
+		automaticExecutionsNumber,
+		c.config.NetworkType,
+	)
+	if tx != nil {
+		c.modifyTransaction(tx)
+	}
+
+	return tx, err
+}
+
+func (c *Client) NewManualCallTransaction(
+	deadline *Deadline,
+	contractKey *PublicAccount,
+	executionCallPayment Amount,
+	downloadCallPayment Amount,
+	fileName string,
+	functionName string,
+	actualArguments string,
```

Comment:
`actualArguments` is `string` or `byte array`? Check, please.

Reply:
https://github.com/proximax-storage/go-xpx-chain-sdk/commit/d65df18c9b8d3f15f43f19350c10a2291a0271f4